src/d/actor/d_a_tpota.cpp

```diff
@@ -5,71 +5,171 @@
 
 #include "d/actor/d_a_tpota.h"
 #include "d/d_procname.h"
+#include "JSystem/JGeometry.h"
 
+#include "JSystem/JParticle/JPAParticle.h"
+#include "d/d_particle.h"
+#include "d/d_com_inf_game.h"
+
+
+const u16 l_daTpota_idx_table[2] = {0x82AE, 0x82AF};
 /* 00000078-000001D0       .text _create__9daTpota_cFv */
 cPhs_State daTpota_c::_create() {
-    /* Nonmatching */
+    fopAcM_SetupActor(this, daTpota_c);
+    int count = 0;
+    JPABaseEmitter* emitter;
+
+    do {
+        position[count] = current.pos;
+        angle[count] = current.angle;
+    
+        emitter = dComIfGp_particle_set(
+            l_daTpota_idx_table[count],
+            &position[count], 
+            &angle[count], 
+            NULL, 0xff, NULL, -1, NULL, NULL, NULL
+        );
+        emitters[count] = emitter;
+        count++;
+    }while (count < 2);
+    return 4;
```

Comment:
```suggestion
    return cPhs_COMPLEATE_e;
```
Create func returns a phase state